Assets/Scripts/StatePattern/ViewRecipeListState.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Written by Ellie McDonald
+// 4/13/21
+
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ViewRecipeListState : MonoBehaviour, State
+{
+    //Dictionary holds the booleans for displaying the crafting and inventory canvas
+    private Dictionary<string, bool> canvasCriteria = new Dictionary<string, bool>();
+    public Dictionary<string, bool> ChooseCraft(InGameDisplay context)
+    {
+        //User is viewing the recipes, until they click back
+        canvasCriteria["craftingDisplay"] = false;
+        canvasCriteria["inventoryDisplay"] = false;
+        canvasCriteria["recipeDisplay"] = false;
+
+        return canvasCriteria;
+    }
+
+    public Dictionary<string, bool> ChooseInventory(InGameDisplay context)
+    {
+        context.State = this.gameObject.GetComponent<InventoryState>();
+        canvasCriteria["craftingDisplay"] = false;
+        canvasCriteria["inventoryDisplay"] = false;
+        canvasCriteria["recipeDisplay"] = false;
+
+        return canvasCriteria;
+    }
+
+    public Dictionary<string, bool> GatherPlant(InGameDisplay context)
+    {
+        //User is viewing the recipes, until they click back
```

Comment:
Delete comment here. The user can walk away and gather plants and the canvas will be disabled, but the user will still be in this state if they don't click back.